Documentation/botr/readytorun-format.md

```diff
@@ -114,6 +112,13 @@ enum ReadyToRunSectionType
     READYTORUN_SECTION_EXCEPTION_INFO               = 104,
```

Comment:
Typo ~10 lines above: `EADYTORUN_SECTION`

Reply:
Thanks for spotting, fixed in 2nd commit.